hassio/src/dialogs/network/dialog-hassio-network.ts

```diff
@@ -107,11 +108,11 @@ export class DialogHassioNetwork extends LitElement
               <ha-svg-icon .path=${mdiClose}></ha-svg-icon>
             </mwc-icon-button>
           </ha-header-bar>
-          ${this._network.length > 1
+          ${this._interfaces.length > 1
```

Comment:
`this._interfaces` can be null if `params.network` was falsey in `showDialog` ?

Reply:
If we have no interfaces, the user does not get the option to open the dialog.